### PR TITLE
Add Message box

### PR DESCRIPTION
**Pull request by hapo-hoangtv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <meta content="width=device-width, initial-scale=1.0" name="viewport">
     <title>Home</title>
     <link href="style.css" rel="stylesheet">
-    <link href="font/OpenSans-Bold.ttf" rel="stylesheet">
-    <link href="font/UTM BryantLG.ttf" rel="stylesheet">
     <link href="fontawesome-free-5.14.0-web/css/all.min.css" rel="stylesheet">
+    <script src="js/jquery-3.5.1.min.js">
```
Comment (hapo-nghialt) [inline index.html]: file script k để đầu

File: index.html [hunk 2]
```diff
@@ -236,13 +257,21 @@ <h3>16,882</h3>
             </div>
         </div>
         <div class="copy-right">
-            &copy; 2020 HapoLearn, Inc. All rights reserved.
+            &copy; 2020 SoloLearn, Inc. All rights reserved.
```
Comment (hapo-nghialt) [inline index.html]: HapoLearn

File: index.html [hunk 3]
```diff
@@ -236,13 +257,21 @@ <h3>16,882</h3>
...
         </div>
     </footer>
     <!-- //footer -->
+    <script>
```
Comment (hapo-nghialt) [inline index.html]: file js tạo file riêng

File: style.css [hunk 4]
```diff
@@ -514,11 +540,123 @@ body {
...
+  margin-top:10px;
+  float: right;
+  cursor: pointer;
+  background-color: #2774DF;
+  border-radius: 50%;
+}
+
+.message-boxchat.active {
+  display: block;
+}
```
Comment (hapo-nghialt) [inline style.css]: EOF

File: style.css [hunk 5]
```diff
@@ -514,11 +540,123 @@ body {
...
+  bottom: 20px;
+  right: 20px;
+}
+
+.message-boxchat {
+  display: flex;
+  height: 190px;
+  flex-direction: column;
+  background-color: white;
+  box-shadow: 0px 0px 8px rgba(0, 0, 0, 0.25);
```
Comment (hapo-nghialt) [inline style.css]: `0px` -> `0`

**Review by hapo-nghialt — APPROVED** [review]
LGTM

File: style.css [hunk 6]
```diff
@@ -2,13 +2,26 @@ html,
...
 
 * {
   box-sizing: border-box;
 }
 
+@font-face {
+  font-family: "UTM BryantLG";
+  font-style: normal;
+  font-weight: normal;
+  src:url("./font/UTM\ BryantLG.ttf");
```
Comment (gs-chinhnc) [inline style.css:16]: dùng [shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#:~:text=Shorthand%20properties%20are%20CSS%20properties,sheets%2C%20saving%20time%20and%20energy.)

File: style.css [hunk 7]
```diff
@@ -2,13 +2,26 @@ html,
...
+}
+
+@font-face {
+  font-family: "Open Sans" sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  src:url("./font/OpenSans-Bold.ttf");
```
Comment (gs-chinhnc) [inline style.css:23]: shorthand

File: style.css [hunk 8]
```diff
@@ -260,16 +279,18 @@ body {
...
   font-family: UTM BryantLG;
   font-style: normal;
+  margin-bottom: 30px;
 }
 
-.hapo-title h1 {
+.hapo-title span {
   font-weight: bold;
   font-size: 80px;
   line-height: 98px;
```
Comment (gs-chinhnc) [inline style.css:285]: shorthand

File: style.css [hunk 9]
```diff
@@ -332,7 +354,7 @@ body {
   max-width: 50%;
 }
 
-.question-why h2 {
+.question-why span {
   font-weight: bold;
   font-size: 70px;
   line-height: 86px;
```
Comment (gs-chinhnc) [inline style.css:356]: chuyển hết font style về shorthand

File: style.css [hunk 10]
```diff
@@ -67,7 +68,7 @@ body {
 
 .footer {
   color: white;
-  font-family: UTM BryantLG;
+  /* font-family: UTM BryantLG; */
```
Comment (hapo-chidl) [inline style.css]: xóa cmt đi nhé

Comment (hapo-hoangtv) [reply style.css]: ok đã xóa

**Review by hapo-chidl — APPROVED** [review]
LGTM

**Pull request merged**